### ISSUE TITLE
DataTable sort & selection stops working on Vue 3.0.1

### ISSUE THREAD

**Issue by LadislavBohm** [primefaces/primevue#582]
With no other changes after updating to Vue 3.0.1 DataTable sorting and selection stopped working. When I click a row nothing gets selected. When I click a header nothing happens as well.

**Comment by dheimoz** [comment]
Thanks for the heads-up LadislavBohm. If you return to ver 3.0.0 does everything work as expected? I saw there were about 9 vulnerabilities related to outdated packages. I created a pull request with the updated packaged. Will give it a try the DataTable Sort with those changes applied and Vue 3.0.1 and comment.

**Comment by dheimoz** [comment]
Hello @cagataycivici I have confirmed the suspicious of LadislavBohm. Vue 3.0.1 breaks VuePrime 3. Not only DataTable sort but also Tree and TreeView.

I believe guys there are a lot of refactor work to be done with Vue 3.0.1 

Will try to narrow the problem. Let you know if I found something useful.

Once again thanks for the great work.

**Comment by cagataycivici** [comment]
3.0.1 spring begins tomorrow, let me work on this, I hope it is nothing serious.

**Comment by dheimoz** [comment]
Thanks for jumping in @cagataycivici . PrimeVue is promising.

**Comment by cagataycivici** [comment]
Could you please test with Vue 3.0.2 as well?

https://github.com/vuejs/vue-next/releases/tag/v3.0.2

**Comment by LadislavBohm** [comment]
Warnings from 3.0.1+ (even 3.0.2 that I tested just now)
![image](https://user-images.githubusercontent.com/4920886/96687169-cc0b5080-137f-11eb-8f37-6baae52310b2.png)

Warnings from 3.0.0
![image](https://user-images.githubusercontent.com/4920886/96687200-da596c80-137f-11eb-9d7d-d8d38f432b13.png)

It looks like update:selection event is not being emitted and v-model:selection is thus not working.

**Comment by cagataycivici** [comment]
Warnings are easy to fix, we'll add the emits: prop, if this is related to it, then should be an easy fix, let me work on it today and get back to you.

**Comment by cagataycivici** [comment]
I'm not able to replicate the sort select issue on table, tried both on 3.0.1 and 3.0.2, seems to work. Can you provide a code sandbox that replicates it? I'll fix the emits warnings now about 549.

**Comment by LadislavBohm** [comment]
Ok I think I found the issue as I was **not** able to reproduce this on any new project I just created. I deleted node_modules and package-lock.json and now it's working fine on Vue 3.0.2 and all warning are gone. Sorry about that.

It's most likely because I bootstrapped the project using vue-cli but some older version when Vue 3 was still in RC phase. Now when you delete package-lock.json it gets a new one and it works. Probably fixes #549 as well.
